src/main.ts

```diff
@@ -10,4 +10,6 @@ if (environment.production) {
 
 platformBrowserDynamic()
   .bootstrapModule(AppModule)
-  .catch(err => console.error(err))
+  // console.error is used here since dependency injection is set up at this moment.
```

Comment:
Shouldn't this be: `is not setup`?

Reply:
Minor thing: Consider placing the comment on the same line as what you are commenting (as long as it is short enough).
So it would be something like:
`.catch(error => ...) // Comment here`

This makes it less likely that the comment will be separated from the code that it is commenting on later in the lifespan of the project.

Reply:
The reason it was on two lines, was that I had trouble getting the comment and the rule on the same line. I found a way to do that, so it is on the same line now.